account-kit/smart-contracts/src/light-account/clients/client.ts

```diff
@@ -67,7 +86,7 @@ export function createLightAccountClient<
  */
 export async function createLightAccountClient(
   params: CreateLightAccountClientParams
-): Promise<SmartAccountClient> {
+): Promise<SmartAccountClient | AlchemySmartAccountClient> {
   const { transport, chain } = params;
 
```

Comment:
I originally called something like this
But the code in 
```suggestion

  if (isAlchemyTransport(transport)) {
    return await createLightAccountAlchemyClient({
      ...params,
      transport,
    });
  }

```
Since the code is the same?
Here is the snapshot of current alchemy client, I don't see a difference
![image](https://github.com/user-attachments/assets/294fd944-3aec-4da6-bc29-8989009c0353)

Reply:
oh the difference is that this by default calls `createSmartAccountClient` and the alchemy calls `createAlchemySmartAccountClient` they take slightly different config params (createSCAClient takes in a `paymasterMiddleware` and createAlchemySCAClient takes in a `gasManagerConfig` or I think just a `policyId`)

Reply:
Okay, I'll go down the side of testing the params to check the transport as alchemyTransport